cellmlmanip/mathml2sympy/transpiler.py

```diff
@@ -3,31 +3,32 @@
 Content Markup specification: https://www.w3.org/TR/MathML2/chapter4.html
 """
 import logging
-from typing import Dict, List
 from xml.dom import Node, minidom
 
 import sympy
 
 
 logger = logging.getLogger(__name__)
-logger.setLevel(logging.INFO)
 
 
 class Transpiler(object):
-    """Transpiler class handles conversion of MathmL to Sympy exprerssions"""
+    """
+    Handles conversion of MathmL to Sympy exprerssions.
 
-    def __init__(self,
-                 dummify: bool = False,
-                 symbol_prefix: str = None,
-                 symbol_lookup=dict()) -> None:
+    :param dummify: A bool that ???
```

Comment:
Because some bits were written a while ago when Asif was still figuring out CellML, and concentrating on getting an initial working 'tracer bullet' to achieve Web Lab project goals, rather than writing libCellML in Python...